lms/templates/static_templates/400.html

```diff
@@ -0,0 +1,18 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Bad Request (400)")}</%block>
+
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="outside-app">
+        <h1>${_("Bad Request (400)")}
+            <br>
+            <a href="javascript:history.back()">Go Back</a>
```

Comment:
The IMS standard requires an invalid launch to take the user to a local error page that includes a link to return back to the tool consumer, so this is my link to return to the tool consumer. It works on the IMS test suite and brings the user back to the page the launch came from, but I'm not sure how to check it works with other consumers?